testsuite/src/main/java/io/netty/testsuite/transport/socket/SocketReadPendingTest.java

```diff
@@ -40,7 +40,7 @@
 import static org.junit.Assert.assertTrue;
 
 public class SocketReadPendingTest extends AbstractSocketTest {
-    @Test(timeout = 30000)
+    @Test(timeout = 60000)
```

Comment:
revert ?

Reply:
No, see the commit message. This test goes through more cases now.